snuba/cli/__init__.py

```diff
@@ -1,51 +1,67 @@
+from __future__ import annotations
+
 import logging
+import os
 import time
-from pkgutil import walk_packages
+from typing import Any
 
 import click
 import sentry_sdk
 import structlog
 
+from snuba.core import initialize
 from snuba.environment import metrics as environment_metrics
 from snuba.environment import setup_logging, setup_sentry
 from snuba.utils.metrics.wrapper import MetricsWrapper
 
 setup_sentry()
-setup_logging("DEBUG")
-logger = logging.getLogger("snuba_init")
 
-start = time.perf_counter()
-logger.info("Initializing Snuba...")
-metrics = MetricsWrapper(environment_metrics, "cli")
+with sentry_sdk.start_transaction(
+    op="snuba_init", name="Snuba CLI Initialization", sampled=True
+):
 
+    setup_logging("DEBUG")
+    logger = logging.getLogger("snuba_init")
 
-@click.group()
-@click.version_option()
```

Comment:
im not too too familiar with click but do we not need these anymore?

Reply:
We don't need `group` because we have switched main to a `MultiCommand` but I can put the version option back in